MeshLib/PropertyVector.h

```diff
@@ -31,6 +31,23 @@ class PropertyVectorBase
 		std::vector<std::size_t> const& exclude_positions
 	) const = 0;
 	virtual ~PropertyVectorBase() = default;
+
+	MeshItemType getMeshItemType() const { return _mesh_item_type; }
+	std::string const& getPropertyName() const { return _property_name; }
+	std::size_t getNumberOfComponents() const { return _n_components; }
+
+protected:
+	explicit PropertyVectorBase(std::string const& property_name,
```

Comment:
no need to set explicit keyword, because the constructor requires three arguments